### PR TITLE
add packages for fullCalendar react

### PR DESCRIPTION
**Pull request by SuyashSalampuria**

### CODE REVIEW

**Comment by algomaster99** [comment]
Will merge this once https://github.com/IMGIITRoorkee/omniport-backend/pull/66 is merged.

File: omniport/core/public/index.html [hunk 1]
```diff
@@ -30,6 +30,10 @@
   <link rel="stylesheet" type="text/css" href="/static/semantic-ui/semantic.min.css" />
   <link rel="stylesheet" type="text/css" href="/static/semantic-ui/semantic.fix.css" />
 
+  <link rel="stylesheet" type="text/css" href="/static/full-calendar-semantic-ui/daygrid.main.css" />
+  <link rel="stylesheet" type="text/css" href="/static/full-calendar-semantic-ui/core.main.css" />
+  <link rel="stylesheet" type="text/css" href="/static/full-calendar-semantic-ui/semantic.fullCalendar.css" />
```
Comment (dhruvkb) [inline omniport/core/public/index.html:35]: So you need to install `@fullcalendar` packages from `npm` **_and_** `link` to stylesheets?

Comment (dhruvkb) [reply omniport/core/public/index.html:35]: @algomaster99 @SuyashSalampuria

Comment (algomaster99) [reply omniport/core/public/index.html:35]: Maybe we could get around this. @SuyashSalampuria just found a resource that could help.

**Pull request merged**